internal/blueprint/blueprint.go

```diff
@@ -43,13 +43,13 @@ type Group struct {
 func (b *Blueprint) DeepCopy() (Blueprint, error) {
 	bpJSON, err := json.Marshal(b)
 	if err != nil {
-		return Blueprint{}, err
+		panic(err)
```

Comment:
I can't say what the implications of this are, maybe @bcl can. However I'm looking at this statement: 
"The return can't be easily covered the way it's written now"

Can you explain why ? 

What is preventing you from supplying appropriate input data (or mocking json.Marshall so it returns an error) then reading the returned values from this function and comparing the result via asserts, e.g. assert.NotNil(t, returned_err) ?

Reply:
Well it only accepts type `Blueprint` as an input and I can't find a way to give it wrong data. It's defined in a way that it only accepts the specified data types:
```
type Blueprint struct {
	Name           string          `json:"name" toml:"name"`
	Description    string          `json:"description" toml:"description"`
	Version        string          `json:"version,omitempty" toml:"version,omitempty"`
	Packages       []Package       `json:"packages" toml:"packages"`
	Modules        []Package       `json:"modules" toml:"modules"`
	Groups         []Group         `json:"groups" toml:"groups"`
	Customizations *Customizations `json:"customizations,omitempty" toml:"customizations,omitempty"`
}
```
so the `json.Marshal` and `json.Unmarshal` will always get the correct input. If you see a way to make it return an error I would be happy if you can share it.